docs/using-kontena/volumes.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Volume Management
+---
+
+# Volumes
+
+Kontena provides capability to manage persistent data using volumes.
+
+Volumes are supported as first-class citizens in both Kontena Master API and on stack yaml definitions.
```

Comment:
Volumes are supported as first-class objects in the Kontena Master API, which can be referred to from stack YAML definitions.